frontend/isoko/src/features/dashboard/DashboardSlice.ts

```diff
@@ -0,0 +1,60 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Business } from '../../types/GlobalTypes';
+import { fetchBusinessDetails, updateBusinessDetails } from './DashboardAPI';
+
+export interface DashboardState {
+   business: Business | null;
+   analytics: null;
```

Comment:
is this null because we don't have analytics set up fully or is it supposed to be null?

Reply:
Yeah its just null for now since we don't have the analytics stuff set up, but once we figure out what we want that to look like we'll have a different type for it